src/main/pages/che-7/end-user-guide/proc_websocket-troubleshooting.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// websocket-troubleshooting
+
+[id="websocket-troubleshooting_{context}"]
+= WebSocket Troubleshooting
```

Comment:
```suggestion
= Troubleshooting WebSocket connections
```

Reply:
This procedure seems connected to the previous one ("Network requirement"). Perhaps the preceding topic could be made more concise and incorporated into this procedure as prerequisites?